examples/ibm-db2/README.md

```diff
@@ -0,0 +1,49 @@
+# This example shows how to create an IBM Cloud Db2 SaaS and configure connectivity from a VSI
```

Comment:
Change this to "This example shows how to create an instance IBM Db2 SaaS on IBM Cloud and configure connectivity from a VSI"